_component_design/selector-strip.md

```diff
@@ -18,15 +18,81 @@ helix-ui-javascript: false
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use selector strips in the following situations:
+
+- When the selector strip is the only input on a form.
+- When vertical spacing isn't available and you want to make use of horizontal spacing. A selector strip can function similarly to [radios]({{site.baseurl}}/components/radio-buttons.html) (single select strip) or [checkboxes]({{site.baseurl}}/components/checkboxes.html) (multi-select strip).
+- When the user's selection alters the data input options presented on the page. For example, use a selector strip when the user selects credit card as the payment option. The fields on the form are dependent on the selected credit card option.
+- When you want an operation to recur periodically. For example, use a selector strip when the user needs to schedule maintenance by selecting from preset days and times.
+- When you want to provide more visual prominence to a group of options.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a selector strip:
+
+- For single select strips, one option is selected by default.
+- A selector strip should contain no more than seven options. It is important to balance the length of the option labels with the number of options listed.
+- Use short option labels (for example, three words maximum and up to 10 characters) so that the number of options can fit horizontally on a small screen.
+- If they are commonly understood, use abbreviations. For example, on a credit card selector strip the user understands that *AmEx* stands for *American Express*.
+- A selector strip should not wrap.
+- Do not use a selector strip to trigger an action. If you need multiple triggers alongside each other, consider a [button bar]({{site.baseurl}}/components/simple-buttons.html) or a [button set]({{site.baseurl}}/components/button-bar.html).
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### When to use
+## Single select strip specifications
 
-Use a selector strip on forms requiring a user to select from a small group of action. Selector strips function similarly to [radio buttons]({{site.baseurl}}/components/radio-buttons.html) or [check-boxes]({{site.baseurl}}/components/checkboxes.html) depending on the variation used: single-select or multi-select. Selector strips are best used in situations where the selection alters the data options presented on the page, or when seeking to more visual prominence to group a of options.
+Consider the following specifications when you construct a single select strip.
```

Comment:
s/`single select`/`single-select`

Reply:
Also, fix throughout (it's a [compound modifier](https://developer.rackspace.com/docs/style-guide/style/punctuation/#hyphens), and therefore needs a hypen).